src/controllers/ratingController.js

```diff
@@ -39,6 +40,31 @@ class ArticleRate {
       data: { rating }
     });
   }
+
+  /**
+     * Check the environment
+     * @function getArticleRating
+     * @param  {object} req - accept object with user info
+     * @param  {object} res - accept object with user info
+     * @return {json} Returns json object
+     * @static
+     */
+  static async getArticleRating(req, res) {
+    const { articleId } = req.params;
+    const getArticle = await Article.findOne({
+      where: { id: articleId }
+    });
+    if (!getArticle) {
+      return res.status(404).json({ error: 'Article not found' });
+    }
+    const getRate = await Rating.findAll({
+      where: { articleId },
+    });
+    if (!getRate.length) {
+      return res.status(200).json({ message: 'This article has no rates so far' });
```

Comment:
The right term is `rating` not `rate`. It's ok to use it internally to distinguish between the model and the value but for displaying to consumers of the API you have to be grammatically correct

Reply:
Okay. Let me work on that Thanks @mkiterian

Reply:
@mkiterian I've Implemented your feedback